packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/debug/DebugLogging.kt

```diff
@@ -0,0 +1,77 @@
+package expo.modules.notifications.notifications.debug
+
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import androidx.annotation.RequiresApi
+import com.google.firebase.messaging.RemoteMessage
+import expo.modules.notifications.BuildConfig
+import expo.modules.notifications.notifications.model.Notification
+import java.util.function.Consumer
+
+object DebugLogging {
+  @JvmStatic
+  fun logBundle(caller: String, bundleToLog: Bundle) {
+    Log.i("ReactNativeJS", "$caller:")
+    logBundle(caller, bundleToLog, "  ")
+  }
+
+  private fun logBundle(ignoredCaller: String, bundleToLog: Bundle, indent: String) {
+    if (!BuildConfig.DEBUG) {
+      // Do not log in release/production builds
+      return
+    }
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N) {
+      return
+    }
+    if (indent == "        ") {
+      return
+    }
```

Comment:
Not a big deal but could be easy for someone to break this by mistake when using the raw string like this

```kotlin
val str = buildString {
    append("\"")
    for (i in 0..5) {
        append(" ")
    }
    append("\"")
}
```
Maybe overkill for something this small so I don't mind if you want to leave it as is

Reply:
I'll take a look... I think the spotlessApply task did that 😄

Reply:
Which file is that in? I can't find it in expo-notifications Android source

Reply:
It's not clear to me how a string like the above would break this, as the function that takes `indent` is not exposed outside this class. Maybe I can avoid this, and make the flow more clear, by using a numerical parameter to indicate recursion depth...

Reply:
Fixed.

Reply:
I was more thinking of developer error rather than usage 😅